### PR TITLE
Add new miner pools to miner domain lists to improve detection 

### PR DESCRIPTION
**Pull request by AlbertoPellitteri**
Signed-off-by: Alberto Pellitteri <albertopellitteri96@gmail.com>
Co-authored-by: darryk10 <stefano.chierici@sysdig.com>

**What type of PR is this?**

/kind rule-update

**Any specific area of the project related to this PR?**

/area rules

**What this PR does / why we need it**:

Found and added some new miner pools that enhance miner detection.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**: 

```release-note
rule(list miner_domains): add new miner domains
rule(list https_miner_domains): add new miner domains
```

### CODE REVIEW

File: rules/falco_rules.yaml [hunk 1]
```diff
@@ -2764,7 +2769,12 @@
     "stratum-ltc.antpool.com",
     "stratum-zec.antpool.com",
     "stratum.antpool.com",
-    "xmr.crypto-pool.fr"
+    "xmr.crypto-pool.fr",
+    "ss.antpool.com",
+    "stratum-dash.antpool.com.antpool.com",
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2774]: should `stratum-dash.antpool.com.antpool.com` be `stratum-dash.antpool.com`

Comment (AlbertoPellitteri) [reply rules/falco_rules.yaml]: Thank you. Typo fixed.

**Review by Kaizhe — APPROVED** [review]

**Review by leogr — APPROVED** [review]
Thank you!
/approve

**Pull request merged**